packages/eas-cli/src/build/utils/analytics.ts

```diff
@@ -18,6 +18,8 @@ export enum Event {
   CREDENTIALS_SYNC_UPDATE_LOCAL_FAIL = 'build cli credentials sync update local fail',
   CREDENTIALS_SYNC_UPDATE_REMOTE_SUCCESS = 'build cli credentials sync update remote success',
   CREDENTIALS_SYNC_UPDATE_REMOTE_FAIL = 'build cli credentials sync update remote fail',
+
+  ACTION = 'action', // generic event type which is used to determine the 'daily active user' stat, include an `action: eas ${subcommand}` property inside of the event properties object
```

Comment:
This enum represents build events, please keep generic event somewhere in root , e.g. in `./src/analytics.ts`
You will also need to use logEvent directly from main analytics.ts for types to work